templates/shared/_cra-banner.html

```diff
@@ -0,0 +1,11 @@
+<section class="p-strip">
```

Comment:
```suggestion
<section class="p-strip u-no-padding--bottom">
```
Personally I would add no padding to the bottom as there is a large gap between the notice and the following content. This also aligns with the usage on the homepage.